docs/pages/blog/2023-toolpad-beta-announcement.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Toolpad is now in Beta!
+description: After working on it for more than a year, Toolpad is now in the next stage. Read more about it in this announcement.
```

Comment:
Let's try to reframe this to center the reader and answer "what's in it for me?"

```suggestion
description: Assemble admin panels and internal tools faster than ever before with Toolpad—now in beta.
```